instaclustr/resource_cluster.go

```diff
@@ -858,3 +1286,35 @@ func checkIfBundleRequiresRackAllocation(bundles []Bundle) bool {
 
 	return true
 }
+
+func isClusterSingleDataCentre(cluster Cluster) bool {
+	if len(cluster.DataCentres) == 1 {
+		return true
+	}
+	return false
+}
+
+func validateMultiDCProvisioningAPI(request CreateRequest) error {
```

Comment:
[question] I don't love this name, would something like `validateMultiDCProvisioningRequest` be better?

Reply:
resolved.